rustsec/Cargo.toml

```diff
@@ -28,10 +28,18 @@ cargo-edit = { version = "0.9", package = "cargo-edit-9", optional = true, defau
     "upgrade",
 ] }
 tame-index = { version = "0.3", features = ["git", "sparse"], optional = true }
-git2 = { version = "0.16.1", optional = true }
 home = { version = "0.5", optional = true }
-humantime = { version = "2", optional = true }
-humantime-serde = { version = "1", optional = true }
+time = { version = "0.3", default-features = false, features = ["formatting", "serde"], optional = true }
```

Comment:
`time` is already a dependency due to `gix`, and has the formatting stuff for serde already so no need for the humantime stuff.